Exception in Java verticle script 
java.lang.UnsupportedOperationException
  at java.util.AbstractMap.put(AbstractMap.java:203)
  at org.vertx.java.core.json.JsonObject.putObject(JsonObject.java:70)
  at jsonrpc.core.JsonRpcResponse$.makeResult(JsonRpcResponse.scala:41)
  at jsonrpc.core.Server$$anonfun$start$2$$anonfun$apply$2$$anonfun$apply$3$$anonfun$apply$5.apply(Server.scala:69)
  at jsonrpc.core.Server$$anonfun$start$2$$anonfun$apply$2$$anonfun$apply$3$$anonfun$apply$5.apply(Server.scala:65)
  at jsonrpc.core.VertxAdaptor$$anon$2.handle(VertxAdaptor.scala:20)
  at org.vertx.java.core.eventbus.impl.DefaultEventBus$7.run(DefaultEventBus.java:592)
  at org.vertx.java.core.impl.Context$2.run(Context.java:119)
  at org.jboss.netty.channel.socket.nio.AbstractNioWorker.processEventQueue(AbstractNioWorker.java:454)
  at org.jboss.netty.channel.socket.nio.AbstractNioWorker.run(AbstractNioWorker.java:331)
  at org.jboss.netty.channel.socket.nio.NioWorker.run(NioWorker.java:35)
  at org.jboss.netty.util.ThreadRenamingRunnable.run(ThreadRenamingRunnable.java:102)
  at org.jboss.netty.util.internal.DeadLockProofWorker$1.run(DeadLockProofWorker.java:42)
  at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
  at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
  at java.lang.Thread.run(Thread.java:722)
